velox/expression/Expr.cpp

```diff
@@ -662,8 +662,7 @@ void Expr::addNulls(
     // Not all Vectors support resize.  Since we only want to append nulls,
     // we can work around that by calling setSize to resize the vector and
     // ensureNullsCapacity to resize the nulls_ bit vector.
-    result->setSize(rows.end());
-    result->ensureNullsCapacity(rows.end(), true);
+    result->resize(rows.end(), true);
```

Comment:
Nice. ensureNullsCapacity is not used anymore outside of the BaseVector, right? Let's make it private or fold it into the caller.

To simplify this PR, consider extracting this change + adding DictionaryVector::resize + test into a separate PR.